lib/galaxy/webapps/galaxy/api/user_preferences.py

```diff
@@ -783,14 +798,7 @@ def get_filter_mapping(db_filters, config_filters, factory):
             }
 
     @expose_api
-    def toolbox_filters(self, trans, cntrller='user_preferences', **kwd):
-        """
-        API call for fetching toolbox filters data
-        """
-        return self.tool_filters(trans, **kwd)
-
-    @expose_api
-    def edit_toolbox_filters(self, trans, cntrller='user_preferences', **kwd):
+    def __edit_toolbox_filters(self, trans, cntrller='user_preferences', **kwd):
```

Comment:
This is a work in progress and in transition. Edit toolbox filters e.g. should either be part of change_toolbox_filters, or split into set/get_toolbox_filters. If change_toolbox_filters does not sound good we can change the name in a consistent fashion.